userguide/sharing.rst

```diff
@@ -755,7 +758,7 @@ best results, use :ref:`Windows (SMB) Shares`.
 From macOS
 ^^^^^^^^^^^^^
 
-To mount the NFS volume from a macOS client, click on
+To mount the NFS volume from a macOS client, from the Finder click on
```

Comment:
"from a macOS client, from the Finder" sounds redundant, even if it isn't.  Sentence probably could be rearranged to avoid that.

Reply:
OK, I copied the wording from the AFP section.